web/static/js/components/category_column.jsx

```diff
@@ -13,7 +13,7 @@ const CategoryColumn = props => {
   return (
     <section className={`${category} ${styles.index} column`}>
       <div className={` ${styles.columnHead} ui center aligned basic segment`}>
-        <img src={`/images/${category}.svg`} height={iconHeight} width={iconHeight} />
+        <img src={`/images/${category}.svg`} height={iconHeight} width={iconHeight} alt={`${category}`} />
```

Comment:
no need to use the backtics with interpolation here. just put `category` between curlys (curlies?)